liquibase-core/src/main/java/liquibase/parser/core/yaml/YamlSnapshotParser.java

```diff
@@ -11,19 +11,25 @@
 import liquibase.resource.ResourceAccessor;
 import liquibase.snapshot.DatabaseSnapshot;
 import liquibase.snapshot.RestoredDatabaseSnapshot;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.LoaderOptions;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.SafeConstructor;
+import org.yaml.snakeyaml.representer.Representer;
+import org.yaml.snakeyaml.resolver.Resolver;
 
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.Map;
 
 public class YamlSnapshotParser extends YamlParser implements SnapshotParser {
 
+    public static final int CODE_POINT_LIMIT = 9 * 1024 * 1024;
```

Comment:
Why not just make this MAX_VALUE? I see no reason to limit the size on our end.

Reply:
Agreed.